### PR TITLE
Small patch due to changes in GenericConstantMaterial in MOOSE

### PR DESCRIPTION
**Pull request by YaqiWang**
Tag @lindsayad .

### CODE REVIEW

File: src/materials/GenericMoltresMaterial.C [hunk 1]
```diff
@@ -43,6 +43,10 @@ validParams<GenericMoltresMaterial>()
                         "greater than the peak power density set point, "
                         "the absorption cross section gets incremented by "
                         "this amount");
+
+  // the following two lines esentially make the two parameters unrequired
```
Comment (katyhuff) [inline src/materials/GenericMoltresMaterial.C]:
```suggestion
  // the following two lines esentially make the two parameters optional
```

**Comment by katyhuff** [comment]
I approve this, but I suppose we should wait for the tests to pass?
(Also, why does it say "Developer needed to activate"? That's new to me.)

**Comment by katyhuff** [comment]
nevermind, answered my own question about dev needing to activate!

**Comment by YaqiWang** [comment]
This change does not need a MOOSE update. But the two parameters will be required for fixing https://github.com/idaholab/moose/issues/13601.

**Pull request merged**